modules/mobile/app/models/mobile/v0/adapters/lighthouse_direct_deposit.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Adapters
+      class LighthouseDirectDeposit
+        def parse(direct_deposit_info)
+          JSON.parse({
+            control_information: parse_control_information(direct_deposit_info.control_information),
+            payment_account: parse_payment_account(direct_deposit_info.payment_account)
+          }.to_json,
+                     object_class: OpenStruct)
```

Comment:
This spacing seems weird. Not sure why the linter doesn't complain.

Reply:
The linter made it that way, I kept trying to line it up properly